components/case-studies/CaseStudies.tsx

```diff
@@ -9,19 +9,22 @@ import { BoxIconContainer, CaseStudiesContainer } from './styled-components';
 
 interface CaseStudiesProps {
   data: Array<CaseStudy>;
+  secondary?: boolean;
 }
 
-const CaseStudies: FC<CaseStudiesProps> = ({ data }) => {
+const CaseStudies: FC<CaseStudiesProps> = ({ data, secondary }) => {
   return (
     <CaseStudiesContainer id="caseStudies">
       {data.map((caseStudy, caseStudyIndex) => {
         const direction = caseStudyIndex % 2 === 0 ? 'row-reverse' : 'row';
 
         return <CaseStudyInner key={caseStudy.id} data={caseStudy} direction={direction} />;
       })}
-      <BoxIconContainer>
-        <RectangleBoxIcon />
-      </BoxIconContainer>
+      {!secondary ? (
```

Comment:
I am unsure if we need this check + additional prop. Let's discuss over the call